test/Libraries/RevitNodesTests/Elements/RoofTypeTests.cs

```diff
@@ -26,16 +28,16 @@ public void CanGetRoofTypeThermalProperties()
             var thermalProperties = roofType.GetThermalProperties();
             double resultRoofTypeAbsorptance = (double)thermalProperties["Absorptance"];
             double resultRoofTypeHeatTransferCoefficient = (double)thermalProperties["HeatTransferCoefficient"];
-            double resultRoofTypeRoughness = (double)thermalProperties["Roughness"];
+            double resultRoofTypeRoughness = (int)thermalProperties["Roughness"];
```

Comment:
type of variable is still `double`, so it would cast the `int` back to `double` again